gdx/src/com/badlogic/gdx/InputMultiplexer.java

```diff
@@ -55,24 +72,33 @@ public int size () {
 		return processors.size;
 	}
 
+	/** Empty the list of processors */
 	public void clear () {
 		processors.clear();
 	}
 
+	/** Empty the list of processors and implement a new list of processors
```

Comment:
OK... this would be a big review.

This line isn't even right. Like, the only part that's correct is that it empties the... well, not a list... of processors. And like the rest of the PR, there's no benefit to `@param processors`, we can see that immediately already. It might help to know it takes 0 or more InputProcessor items as varargs or in an array! Just knowing its name doesn't tell me that.